src/controllers/itemdetailpage.js

```diff
@@ -521,11 +521,7 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "cardBuild
         renderDetails(page, item, apiClient, context);
         renderTrackSelections(page, instance, item);
 
-        if (dom.getWindowSize().innerWidth >= 1000) {
-            backdrop.setBackdrops([item]);
-        } else {
-            backdrop.clear();
-        }
+        backdrop.clear();
```

Comment:
Why not just check the backdrop setting here as well for the time being? We can combine the code later but at least then it would be configurable if people like the backgrounds.

Reply:
i am thinking on detailpage like plex

Reply:
![detail](https://user-images.githubusercontent.com/32230989/69583786-bfa23a80-0fec-11ea-8db1-b61b35384f70.JPG)
![detail-2](https://user-images.githubusercontent.com/32230989/69583825-d0eb4700-0fec-11ea-8c10-b3534adc6a22.JPG)

Reply:
I wonder if we need the backdrops at all if we go that route. Not sure how many people actually like them but I honestly prefer the Plex interface a lot better, which has no shaded backdrops.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/594/commits/7afdc4b2b1dd9f3f5f98342a434ee9a225a76b59